code/script.js

```diff
@@ -0,0 +1,170 @@
+                data.filter(repo => {
+                    return repo.fork && repo.name.startsWith("project-") && selectedLanguages.includes(repo.language)
+                })
+            //Function to sort repos by latest date.
+            forkedRepos.sort(function (a, b) {
+                return new Date(b.pushed_at) - new Date(a.pushed_at)
+            })
+            forkedRepos.forEach(repo => {
+                projectsContainer.innerHTML += `
+                      <div class="repo" id=${repo.name}>
+                        <p class="forked-repos"> <img src="${repoIcon.src}" alt="${repoIcon.alt}" class="repo-icon"/> ${repo.name} </p> 
+                        <a href="${repo.html_url}" target="blank">Go to repo!</a> 
+                        <p class="forked-repos"> <img src="${branchIcon.src}" alt="${branchIcon.alt}" class="branch-icon"/> ${repo.default_branch}</p>
+                        <p class="forked-repos"> <img src="${pushIcon.src}" alt="${pushIcon.alt}" class="push-icon"/> ${new Date(repo.pushed_at).toDateString()}</p>
+                        <p id="commit-${repo.name}">Amount of commits: </p> 
+                      </div>
+                        `
+            })
+
+            fetchPullRequests(forkedRepos)
+
+            //Only draw the chart once.
+            if (!chartDrawn) {
+                drawChart(forkedRepos.length)
+                chartDrawn = true
+            }
+        })
+}
+
+const fetchPullRequests = (repos) => {
+    repos.forEach(repo => {
+        const PULL_URL = `https://api.github.com/repos/Technigo/${repo.name}/pulls?per_page=100`
+
+        fetch(PULL_URL)
+            .then(res => res.json())
+            .then((data) => {
+                const myPullRequest =
+                    data.find(pull => pull.user.login === repo.owner.login)
+                if (myPullRequest) {
+                    fetchCommits(myPullRequest.commits_url, repo.name);
+                } else {
+                    document.getElementById(`commit-${repo.name}`).innerHTML =
+                        "No pull request yet!";
+                }
+            })
+    })
+}
+
+const fetchCommits = (myCommitsUrl, myRepoName) => {
+    fetch(myCommitsUrl)
+        .then(res => res.json())
+        .then(data => {
+            document.getElementById(`commit-${myRepoName}`).innerHTML += data.length
+        })
+}
+
+const filterLanguages = (event) => {
+    console.log(event)
+    const language = event.target.id
+    const checked = event.target.checked
+
+    console.log(language, checked)
+
+    //Adds language string in array if checked language is missing.
+    if (checked && !selectedLanguages.includes(language)) {
+        selectedLanguages.push(language)
+        //Removes language string in array if unchecked language is in array.  
+    } else if (!checked && selectedLanguages.includes(language)) {
+        for (let i = 0; i < selectedLanguages.length; i++) {
+            console.log("i=", i)
```

Comment:
maybe clean up the code and take away all the console logs? just to make the code clean